api/src/main/scala/org/ensime/api/config.scala

```diff
@@ -3,17 +3,26 @@
 package org.ensime.api
 
 import java.io.File
+import java.nio.file.Files
+
+import scala.collection.JavaConverters._
 
 final case class EnsimeConfig(
-  @deprecating("rootDir is no longer used except in testing") rootDir: File,
-  cacheDir: File,
-  javaHome: File,
-  name: String,
-  scalaVersion: String,
-  @deprecating("each project will have a compiler") compilerArgs: List[String],
-  javaSources: List[File],
-  projects: List[EnsimeProject]
-)
+    @deprecating("rootDir is no longer used except in testing") rootDir: File,
+    cacheDir: File,
+    javaHome: File,
+    name: String,
+    scalaVersion: String,
+    @deprecating("each project will have a compiler") compilerArgs: List[String],
+    javaSources: List[File],
+    projects: List[EnsimeProject]
+) {
+  lazy val javaRunTime: List[File] = tree(javaHome).filter(_.getName == "rt.jar").toList
```

Comment:
Instead of traversing files recursively do we know for a fact where `rt.jar` is located?

`$JAVA_HOME/jre/lib/rt.jar` is this location only for my machine?